src/Geocoder/Provider/GoogleMaps.php

```diff
@@ -153,8 +153,9 @@ private function executeQuery($query)
             throw new NoResult(sprintf('Could not execute query "%s".', $query));
         }
 
-        if ('REQUEST_DENIED' === $json->status && 'The provided API key is invalid.' === $json->error_message) {
-            throw new InvalidCredentials(sprintf('API key is invalid %s', $query));
+        if ('REQUEST_DENIED' === $json->status) {
```

Comment:
Ok, I see. However, the exception is thrown only when the request is denied because of credentials issue, which is not the case anymore with your patch.